src/runtime/MethodBinder.cs

```diff
@@ -271,10 +276,11 @@ internal static int ArgPrecedence(Type t)
         /// <param name="inst">The Python target of the method invocation.</param>
         /// <param name="args">The Python arguments.</param>
         /// <param name="kw">The Python keyword arguments.</param>
+        /// <param name="argsReversed">Reverse arguments of methods. Used for methods such as __radd__, __rsub__, __rmod__ etc</param>
         /// <returns>A Binding if successful.  Otherwise null.</returns>
-        internal Binding? Bind(BorrowedReference inst, BorrowedReference args, BorrowedReference kw)
+        internal Binding? Bind(BorrowedReference inst, BorrowedReference args, BorrowedReference kw, bool argsReversed = false)
```

Comment:
Do you need to pass this parameter at all? Don't you have the field?

Reply:
Cleaned it up a bit. I missed that... The one method is static though so we will need to still pass it there...